h/templates/home.html.jinja2

```diff
@@ -223,7 +223,7 @@
           </div>
         </div>
         <div class="press-release-banner">
-            <p>Hypothesis announces a coalition of over 40 scholarly organizations bringing annotation to all knowledge. <a href="https://hypothes.is/annotating-all-knowledge/"><b>Learn more</b></a></p>
+            <p><a href="https://hypothes.is/blog/elife-partners-w…larly-annotation/">eLife Partners with Hypothes.is to Advance Open Scholarly Annotation.</a></p>
```

Comment:
Oh, and the URL is truncated here. `elife-partners-w…larly-annotation` should be `elife-partners-with-hypothes-is-to-advance-open-scholarly-annotation`.